apps/api/src/applicant/applicant.controller.ts

```diff
@@ -51,7 +55,39 @@ export class ApplicantController {
   @ApiResponse({ status: HttpStatus.OK, type: EmptyResponse })
   @HttpCode(HttpStatus.OK)
   @Get('/:id')
-  getApplicant(@Param('id') id: string): Promise<ApplicantEntity> {
-    return this.applicantService.getApplicantById(id);
+  getApplicant(
+    @Param('id') id: string,
+    @Query() relation: ApplicantFilterByIdRO,
+  ): Promise<ApplicantEntity> {
+    return this.applicantService.getApplicantById(id, relation);
+  }
+
+  @ApiOperation({
+    summary: 'Add Applicant',
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @ApiResponse({ status: HttpStatus.CREATED, type: EmptyResponse })
+  @HttpCode(HttpStatus.CREATED)
+  @Post()
+  async addApplicant(@Body() addApplicant: ApplicantCreateRO): Promise<ApplicantEntity> {
```

Comment:
Nit - We should return and RO instead of an entity so we can limit the fields the client receives. 
Usually the pattern is to fetch the Entity and then call a `toResponseObject` function on the entity.